charts/opensearch/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ---
 ## [Unreleased]
 ### Added
+- Added image definition for `fsgroup-volume` initContainer to `values.yaml`.
```

Comment:
Wrong edit, you should create a 1.3.1 block and add your change notes.
Thanks.

Reply:
My bad. Thank you @peterzhuamazon

Reply:
Thanks.